data/data/bootstrap/files/usr/local/bin/fletcher8

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
```

Comment:
+#!/usr/libexec/platform-python
^^ needs to be changes as of the change to ootpa

Reply:
well, once it goes to an init container of the static pod, it will have to be /usr/bin/env python again. So I'll try it when I rebase it.

Reply:
ah, shucks, this is for kni. yes. I need to rename it then :-)